data_structures/linked_list/add_nodes.py

```diff
@@ -19,20 +20,40 @@ def __init__(self):
         self.head = None
 
     # method to print the linked list
-    def printLL(self):
+    def printLL(self) -> None:
```

Comment:
```suggestion
    def __str__(self) -> str:
```
Then we can just `print(my_linked_list)` as discussed [here](https://docs.python.org/3/reference/datamodel.html#object.__str__).